silta-cluster/Chart.yaml

```diff
@@ -36,7 +36,7 @@ dependencies:
   repository: https://charts.jetstack.io/
   condition: cert-manager.enabled
 - name: silta-proxy
-  version: 0.1.0
+  version: 1.0.0
```

Comment:
Let's start using version ranges instead of pinning to an exact version, for dependencies to our own charts.

So test if `version: ~1.0` works, that should pin only to the major version.

Same goes for the silta-release dependency in the *-project-k8s repo PRs